airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -135,7 +135,7 @@ def request_params(
 
 
 class Events(IncrementalAmplitudeStream):
-    cursor_field = "event_time"
+    cursor_field = "server_upload_time"
```

Comment:
Could you please give more context about why `server_upload_time` is a better cursor?

Reply:
[Amplitude documentation](https://www.docs.developers.amplitude.com/analytics/apis/export-api/#considerations) of the Export API (see: first bullet point under "Considerations") states that the intended cursor field for the Export API is `server_upload_time`:
"The specified date range refers to the time the event data was uploaded to Amplitude servers (see server_upload_time)."

Reply:
Great finding, thank you.